community/kernel/src/main/java/org/neo4j/kernel/impl/api/StateHandlingStatementOperations.java

```diff
@@ -120,6 +119,22 @@ public StateHandlingStatementOperations(
     }
 
     // <Cursors>
+
+    @Override
+    public Cursor<NodeItem> loadNodeById( KernelStatement statement, long nodeId ) throws EntityNotFoundException
```

Comment:
I am unaware of name convention about cursors, what am I missing?

You are probably right those methods should be `protected`. I'll change that.

Btw these "new methods" were basically moved down from `OperationsFacade` and renamed.